samples/highcharts/website/homepage-2023-new/demo.js

```diff
@@ -2329,13 +2324,37 @@ const rb = {
     chart: {
         type: 'column',
         animation: {
-            duration: 2000,
+            duration: 3000,
             easing: 'easeOutQuint'
         },
+        margin: [0, 0, 0, 0],
         inverted: true,
+        spacing: 0,
         polar: true,
-        margin: [10, 0, 0, 0],
-        backgroundColor: 'transparent'
+        backgroundColor: 'transparent',
+        events: {
+            load: function () {
+                const chart = this;
+                console.log(chart.chartWidth);
```

Comment:
This should probably be removed, otherwise it will show up in the console for users visiting our website.

Reply:
Hi, I removed the console.logs I didn't need and commented out the others.